src/test/vat.t.sol

```diff
@@ -645,35 +644,90 @@ contract BiteTest is DSTest {
         assertEq(cat.box(), rad(75 ether));
         assertEq(cat.litter(), 0);
         uint auction = cat.bite("gold", address(this));
-        // (,,,,,,,uint tab) = flip.bids(auction);
-        // assertTrue(rad(75 ether) - cat.litter() < ray(3 ether));
-        // assertEq(cat.litter(), tab);
+
         assertEq(ink("gold", address(this)), 50 ether);
         assertEq(art("gold", address(this)), 75 ether);
         assertEq(vow.sin(now), rad(75 ether));
         assertEq(gem("gold", address(this)), 900 ether);
 
-        assertEq(vat.balanceOf(address(this)), 150 ether);
-        assertEq(vat.balanceOf(address(vow)),    0 ether);
-        flip.tend(auction, 50 ether, rad( 1 ether));
+        assertEq(vat.dai(address(this)),  rad(150 ether));
+        assertEq(vat.dai(address(vow)),     rad(0 ether));
+        flip.tend(auction, 50 ether, rad(1 ether));
         assertEq(cat.litter(), rad(75 ether));
-        assertEq(vat.balanceOf(address(this)), 149 ether);
+        assertEq(vat.dai(address(this)), rad(149 ether));
         flip.tend(auction, 50 ether, rad(75 ether));
-        assertEq(vat.balanceOf(address(this)), 75 ether);
+        assertEq(vat.dai(address(this)), rad(75 ether));
 
         assertEq(gem("gold", address(this)),  900 ether);
         flip.dent(auction, 25 ether, rad(75 ether));
         assertEq(cat.litter(), rad(75 ether));
-        assertEq(vat.balanceOf(address(this)), 75 ether);
+        assertEq(vat.dai(address(this)), rad(75 ether));
         assertEq(gem("gold", address(this)), 925 ether);
         assertEq(vow.sin(now), rad(75 ether));
 
         hevm.warp(now + 4 hours);
         flip.deal(auction);
         assertEq(cat.litter(), 0);
         assertEq(gem("gold", address(this)),  950 ether);
-        assertEq(vat.balanceOf(address(this)), 75 ether);
-        assertEq(vat.balanceOf(address(vow)),  75 ether);
+        assertEq(vat.dai(address(this)),   rad(75 ether));
+        assertEq(vat.dai(address(vow)),    rad(75 ether));
+    }
+
+    // tests a partial lot liquidation because it would fill the literbox
+    function test_partial_litterbox_realistic_values() public {
```

Comment:
Hmmmm if we're being really realistic, we should also have a test with non-unity `rate` as well as `chop`...although I think I do that in at least one `testFail`, so maybe it's technically covered.

Reply:
Added a `rate`